CONTRIBUTING.md

```diff
@@ -6,16 +6,10 @@ This project welcomes contributions and suggestions. Most contributions require
 agree to a Contributor License Agreement (CLA) declaring that you have the right to,
 and actually do, grant us the rights to use your contribution.
 
-For details, visit https://cla.microsoft.com.
-
 When you submit a pull request, a CLA-bot will automatically determine whether you need
```

Comment:
remove this para also

Reply:
I dont see it removed in go sdk PR.

Reply:
It should be removed. As this refers to the CLA bot which will no longer be used.

Reply:
Sure. I will remove it.